libcontainer/apparmor/apparmor_disabled.go

```diff
@@ -1,4 +1,4 @@
-// +build !apparmor !linux
+// +build !linux
```

Comment:
Perhaps this file should be renamed to `_unsupported.go` ?

Reply:
Somewhat orthogonal, but wondering if we should have a single implementation of these;

- rewrite `IsEnabled()` to use a `sync.Once` (could use `runtime.GOOS` to skip the actual check)
- rewrite `ApplyProfile()` to use `IsEnabled()`

Note that behavior is currently somewhat inconsistent;

- on non-linux, we produce an error if  `ApplyProfile()` is called (because `ErrApparmorNotEnabled`)
- on linux, `ApplyProfile()` does _NOT_ check if AppArmor is enabled, and always proceeds with `setProcAttr()` (not sure if that produces an error in that case, or is silently ignored)?

Reply:
Agree WRT renaming to `_unsupported`, the rest of it can be addressed as a follow-up.

Reply:
changed to _unsupported. I agree that the rest can be a follow-up.